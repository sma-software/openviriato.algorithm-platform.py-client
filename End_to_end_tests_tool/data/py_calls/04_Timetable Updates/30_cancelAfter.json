{
    "PythonParameterMapping": {
        "train_id" : "$.urlParameters.[0]",
        "train_path_node_id" : "$.urlParameters.[1]"
    }
}