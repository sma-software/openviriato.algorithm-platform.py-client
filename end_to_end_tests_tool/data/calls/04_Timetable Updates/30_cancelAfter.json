{
    "restResource": "/trains/{trainId}/train-path-nodes/{trainPathNodeId}:cancel-after",
    "httpMethod": "PUT",
    "urlParameters": [ "=cancelTrainFrom:id=", "=cancelTrainFrom:trainPathNodes[1].id=" ],
    "requestBody": null,
    "CSharpMethodName": "CancelTrainAfter(trainID, trainPathNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64" ],
    "subcategory": "Cancel Train After"
}
