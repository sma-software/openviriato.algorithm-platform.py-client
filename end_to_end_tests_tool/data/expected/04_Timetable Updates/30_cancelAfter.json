{
  "id": 8116,
  "trainPathNodes": [
    {
      "id": 8113,
      "sectionTrackId": null,
      "nodeId": 7,
      "nodeTrackId": 8,
      "formationId": 1187,
      "arrivalTime": "2003-09-01T00:10:00",
      "departureTime": "2003-09-01T00:10:00",
      "minimumRunTime": null,
      "minimumStopTime": "P0D",
      "stopStatus": "commercialStop",
      "sequenceNumber": 0
    },
    {
      "id": 8114,
      "sectionTrackId": 1175,
      "nodeId": 15,
      "nodeTrackId": 16,
      "formationId": null,
      "arrivalTime": "2003-09-01T00:14:00",
      "departureTime": "2003-09-01T00:14:00",
      "minimumRunTime": "PT4M",
      "minimumStopTime": "P0D",
      "stopStatus": "passing",
      "sequenceNumber": 1
    }
  ],
  "debugString": "RVZH_12_1_J03 tt_(S)"
}